src/test/java/model/CarsTest.java

```diff
@@ -1,26 +1,26 @@
 package model;
 
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
-
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+
 import java.util.List;
 
-public class CarsTest {
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class CarsTest {
 
     @DisplayName("자동차 이름이 중복되면 에러가 발생한다.")
     @Test
     void nameDuplicatedTest() {
-        List<String> carNames = List.of("lemon","lemon","a","nyang");
-        assertThatThrownBy(() -> RacingGame.from(carNames))
-                .isInstanceOf(IllegalArgumentException.class);
+        final List<String> carNames = List.of("lemon","lemon","a","nyang");
+        assertThatThrownBy(() -> Cars.from(carNames));
```

Comment:
```suggestion
        assertThatThrownBy(() -> Cars.from(carNames)).isInstanceOf(IllegalArgumentException.class);
```
예외가 발생할 거라면 어떤 예외인지 명시해주면 좋겠네요!